charts/opentelemetry-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.57.0
+version: 0.58.0
```

Comment:
Since this is a backwards-compatible feature add bump the patch version instead of the minor (since we don't have a major to use yet we normally save the minor bumps for more impactful changes.)

Reply:
Done!